src/components/pagination/pageination-story.ts

```diff
@@ -30,7 +31,11 @@ export const defaultStory = ({ parameters }) => {
         <option value="20">20</option>
         <option value="30">30</option>
       </bx-page-sizes-select>
-      <bx-pages-select></bx-pages-select>
+      ${total == null
```

Comment:
would this work instead of ternary?

```js
(total === null) && html`<bx-pages-select></bx-pages-select>`
```

Reply:
I think `lit-html` renders `false` if I understand it correctly. Good thought to make it more idiomatic, though.